src/main/java/org/jenkinsci/plugins/workflow/log/DelayBufferedOutputStream.java

```diff
@@ -135,9 +130,17 @@ void flushAndReschedule() {
         }
 
         @Override public void flush() throws IOException {
+            try {
             if (enableFlush.get()) {
```

Comment:
indentation?

Reply:
Intentionally unchanged to keep `git blame` as useful as possible, but I am happy to indent it if preferred.

Reply:
that's what i figured. Nah, i'm fine with it.